prov/rxm/src/rxm_conn.c

```diff
@@ -229,7 +229,7 @@ void *rxm_msg_listener(void *arg)
 			FI_WARN(&rxm_prov, FI_LOG_FABRIC, "Unknown event: %u\n", event);
 		}
 		continue;
-err:
+	err:
```

Comment:
Labels are always indented to the beginning of the line (following coding style in linux kernel).